### DISCUSSION TITLE
Taking a Short Break

### DISCUSSION THREAD

**Discussion by eduardogsilva** [eduardogsilva/routerfleet#32] [General]
Hello Everyone,

I wanted to let you know that I'll be taking a short break and will be away until May 13th. During this period, I'll still be available to respond to questions and discuss issues, but I won't be doing any coding.

Cheers!

**Comment by jakes0ns** [comment]
Still eagerly awaiting your return... 

Thanks for all your existing work :D

**Comment by eduardogsilva** [comment]
Hello @jakes0ns! 

My return took a bit longer than expected due to the floods in the south of Brazil. The airport is closed and I had to wait more than a week in SP to get a new flight to another airport.

I'm still working on the busiest part of another project (hotspot management). When this part is complete, I will have more time for routerfleet and wireguard_webadmin. 

If you need something specific, I can take a look for you.

Cheers!
